content/en/checkout/reference.md

```diff
@@ -15,76 +15,100 @@ Creates a Checkout Session.
 
 #### Parameters:
 
-<!-- TODO: Required Vs Optional, it's unclear -->
-<!-- TODO: Better way to show this! -->
-
 An object with the following properties:
 
-- `type` (*required*): `Array` of `string`s representing which [payment type(s)](/concepts/payment-types) this Checkout session should accept. For now, only `['single']` is supported.
-- `payment` (*required*): `Object` describing the payment configuration, with the following properties:
-  - `methods`: `Array` of `string`s with the payment methods accepted in this Checkout session. The available ones are:
-    - `'cc'` (Credit card)
-    - `'mbw'` (MB WAY)
-    - `'mb'` (Multibanco)
-    - `'dd'` (Direct Debit)
-    - `'vi'` (Virtual IBAN)
-    - `'uf'` (Universo Flex)
-    - `'sc'` (Santander Consumer)
-  - `type`: `string` indicating the type of payment for Credit Card and MB WAY operations. Either `'sale'` (default) or `'authorisation'`.
-  - `capture`: `Object` required if the payment `type` is sale.
+<div class="ep-protocol-list">
+
+- `type`: `Array` of `string`s representing which [payment type(s)](/concepts/payment-types) this Checkout session should accept. Either `['single']` or `['frequent']` are supported. (*required*)
```

Comment:
`Array of string` or `String Array` instead of '`Array of string`s'. We should note the fact that only the first value of the array will count.